### PR TITLE
Adding Azure Pipeline to package on Linux, MacOS, and Windows

### PR DESCRIPTION
**Pull request by blowekamp**
Perform packaging for Linux for manylinux1 Python wheels in Docker
container 32 and 64-bits. Package MacOS wheels just of 64-bits. On
Windows build on Visual Studio for Python 2.7 and Visual Studio 2017
with v140 (VS2015) compiler, for both 32 and 64-bits.

### CODE REVIEW

File: Testing/CI/Azure/azure.cmake [hunk 1]
```diff
@@ -1,11 +1,10 @@
-
 # set_from_env
 # ------------
 #
 # Sets a CMake variable from an environment variable. If the
```
Comment (zivy) [inline Testing/CI/Azure/azure.cmake]: Looks like the original file was correct and the modified one has the grammar issues.

**Review by zivy — COMMENTED** [review]
LGTM, just a strange thing in the azure.cmake where the old file looks good and the new has grammar issue?

File: Testing/CI/Azure/azure.cmake (hunk 1 above)
Comment (blowekamp) [reply Testing/CI/Azure/azure.cmake]: Thanks for detecting some rebasing issues.

**Comment by blowekamp** [comment]
@kaspermarstal You may be interested in this PR>

**Review by zivy — APPROVED** [review]
LGTM.

**Pull request merged**